include/tlog/source.h

```diff
@@ -33,6 +33,7 @@
 /** Abstract source */
 struct tlog_source {
     const struct tlog_source_type *type;   /**< Type */
+    struct timespec last_timestamp;        /**< Timestamp */
```

Comment:
Ah, no I meant like this:

```C
/** Abstract source */
struct tlog_source {
    const struct tlog_source_type  *type;           /**< Type */
    struct timespec                 last_timestamp; /**< Timestamp */
```

Notice how `type` and `last_timestamp` names are aligned on the same column.
Take a look at other similar structure declarations.

Reply:
Oh, and wrap this one in `#ifndef NDEBUG`, and fixup the comment too:

```C
/** Abstract source */
struct tlog_source {
    const struct tlog_source_type  *type;           /**< Type */
#ifndef NDEBUG
    struct timespec                 last_timestamp; /**< Last timestamp */
#endif
```